lib/model/folder_sendrecv.go

```diff
@@ -64,7 +64,7 @@ var (
 	errSymlinksUnsupported = errors.New("symlinks not supported")
 	errDirHasToBeScanned   = errors.New("directory contains unexpected files, scheduling scan")
 	errDirHasIgnored       = errors.New("directory contains ignored files (see ignore documentation for (?d) prefix)")
-	errDirNotEmpty         = errors.New("directory is not empty")
+	errDirNotEmpty         = errors.New("directory is not empty; files within are probably ignored on connected devices only")
```

Comment:
What about just containing files we ignore, but without `(?d)`?

Reply:
That's `errDirHasIgnored` just above.